public/javascripts/SVLabel/src/SVLabel/modal/ModalMissionCompleteProgressBar.js

```diff
@@ -71,25 +71,31 @@ function ModalMissionCompleteProgressBar (uiModalMissionComplete) {
      * @private
      */
     this.update = function (missionDistanceRate, userAuditedDistanceRate, otherAuditedDistanceRate) {
+
+        // Round the rates to 0.01 accuracy.
+        var roundedMissionDistanceRate = parseFloat(missionDistanceRate.toFixed(2));
```

Comment:
This is looking a lot better! I think you actually want to go to 3 decimal places instead of 2. At 2 decimal places, the smallest amount we will show is 0.01, and 0.01 * 760 = 7.6. Which means that the smallest granularity we can go to is 7 pixels. But if you go to 3 places after the decimal, the smallest granularity is 1 pixel (0.001 * 760 = 0.76 ~ 1 pixel).